.appveyor.yml

```diff
@@ -24,6 +23,7 @@ after_build:
 branches:
   only:
     - master
+    - fix-path-printing
```

Comment:
This line should be removed?

Reply:
Oops, forgot to remove that commit. Fixed.